### ISSUE TITLE
Update Q&A related training materials / KBs

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#13575]
### User story
**AS AN** editor
**I WANT** modern training materials that reflect the state of how the CMS works today
**SO THAT** I can accurately edit and create content with confidence in how the CMS works.

### Notes / background
PW has updated the behavior for Single Q&A placement on R&S and FAQ nodes. Training materials have lagged behind.

Existing KBs / docs: 
- [x] https://prod.cms.va.gov/help/cms-basics/how-to-create-and-reuse-qas
- [x] https://prod.cms.va.gov/help/resources-and-support/a-guide-to-resources-and-support-content-types
    * Includes link to an outdated video: https://www.youtube.com/watch?v=fkYrLG1F6uo&ab_channel=VACMSTraining -- need to either EOL this video, recreate it as text in the KB, or recreate the video.

## Acceptance criteria

- [x] Q&A reference content is updated and accurate for state of the CMS today

**Comment by FranECross** [comment]
Jordan to do a quick review of the Q&A things.

**Comment by jilladams** [comment]
I poked through the KB to confirm if anywhere else referenced single Q&As. Made one additional udpate. 

The training video referenced here, https://www.youtube.com/watch?v=fkYrLG1F6uo&ab_channel=VACMSTraining, also has a visual of the single-select Beneficiaries field. We're currently reviewing the possibility of updating that to multi-choice on all places where it appears which includes FAQ, so if we're going to re-record, would be best to do it once and capture both sets of changes. I've added that to the CM ticket for that effort:  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15982.  FYI @davidconlon 

Closing this ticket.
